### ISSUE TITLE
cross-validation and train-test

### ISSUE THREAD

**Issue by NadeemNicoR** [pycaret/pycaret#1838]
Is it not possible to not use cross validation but just train and test with the training and the holdout set and then do tune the hyperparameters also without cross validation (simply put i just want to disable cross validation, i understand the point of having a more stable score by averaging the result that you get by CV but i just want to train with respect to a specific split, then check its performance on the hold outset and then tune the hyperparameters )

**Comment by moezali1** [comment]
@NadeemNicoR In the training functions like `compare_models` or `create_model` just pass `cross_validation=False` and PyCaret will skip cross-validation and the metrics that get printed are based on the test set created or passed during the `setup` stage.

Hope this helps.
